### DISCUSSION TITLE
How customize UI for my needs ?

### DISCUSSION THREAD

**Discussion by pico1220** [solidusio/solidus#5834] [New Admin UI/UX]
Hello, I'm looking how works the admin UI with other gems solidus ? I would like also customize design or layouts. Should I override in my own app folder ? 
Thank you

**Comment by jarednorman** [comment]
Customizing the Admin UI's look isn't something that's officially supported, but there's nothing stopping you from adding your own styles.

**Reply by pico1220** [reply]
I'm very very late... but thank you!
